ui/webui/test/check-storage

```diff
@@ -1045,8 +1045,8 @@ class TestStorageMountPointsEFI(anacondalib.VirtInstallMachineCase):
         s.select_mountpoint_row_device(3, f"{dev}3")
         s.check_mountpoint_row_format_type(3, "xfs")
 
-        # Create user on the first pass through users step
-        i.reach(i.steps.REVIEW, step_callbacks={i.steps.ACCOUNTS: lambda: create_user(self)})
+        self.addCleanup(lambda: dbus_reset_users(self.machine))
```

Comment:
This should be added in anacondalib and be run by default for all tests.

Reply:
Okay, sounds good, only on destructive tests if failed for me.
I think we can address it in a follow up PR.